### PR TITLE
Improve demo

### PR DESCRIPTION
**Pull request by blasten**
screen shot 2015-05-31 at 10 24 42 pm

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -17,22 +17,29 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <link href="../../paper-styles/paper-styles.html" rel="import">
+    <link href="../../paper-styles/demo-pages.html" rel="import">
     <link rel="import" href="../iron-media-query.html">
```
Comment (notwaldorf) [inline demo/index.html]: `rel=import` should go at the beginning (i think i missed this a bunch of times too)

File: demo/index.html [hunk 2]
```diff
@@ -17,22 +17,29 @@
...
-        console.log("ok");
-        output.textContent = 'query-matches changed: ' + e.detail.value;
-      });
+      <template is="dom-if" if="{{queryMatches}}">
+        <p>The viewport’s width is greater than <code>600px</code></p>
+      </template>
 
-    </script>
+      <template is="dom-if" if="{{!queryMatches}}">
+        <p>The viewport’s width is less than <code>600px</code></p>
```
Comment (notwaldorf) [inline demo/index.html]: Oh this is SO much nicer. Now the demo actually makes sense :)

**Comment by notwaldorf** [comment]
lgtm % nit

**Comment by blasten** [comment]
Fixed now.

**Pull request merged**